transforms/images/polus-imagej-image-integral-plugin/plugin.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "ImageJ image integral",
+  "version": "0.3.0",
+  "title": "ImageJ image integral",
+  "description": "This plugin applies the image integral algorithm to an input collection.",
+  "author": "Benjamin Houghton (benjamin.houghton@axleinfo.com)",
```

Comment:
You should include the names of everyone that has contributed to the plugin pipeline. Myself and Anjali Taneja (you can leave her email off of it).

Reply:
This has been updated